bundle/regal/rules/testing/metasyntactic_variable.rego

```diff
@@ -32,6 +32,8 @@ report contains violation if {
 
 	lower(ref.value) in metasyntactic
 
+	not ast.is_chained_rule_body(rule, input.regal.file.lines)
```

Comment:
I feel like some of these early ejects triggered by other rules are worthy of explanatory comments.

Reply:
I wouldn't say it's triggered by another _rule_, just that in this case if you say:

```rego
foo {
    true
} {
    false
}
```
There is only really one "foo" to point at. But I'll add a comment to explain this in the code 👍